python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -324,18 +349,46 @@ class carrying all of the information for the object whose image
                                                                                 gsObject.yPupilRadians,
                                                                                 chipName=detector.name)
 
-                obj = centeredObj.copy()
+                obj = centeredObj
 
                 # convolve the object's shape profile with the spectrum
                 obj = obj.withFlux(gsObject.flux(bandpassName))
 
-                self.detectorImages[name] = obj.drawImage(method='phot',
-                                                          gain=detector.photParams.gain,
-                                                          offset=galsim.PositionD(xPix-detector.xCenterPix,
-                                                                                  yPix-detector.yCenterPix),
-                                                          rng=self._rng,
-                                                          image=self.detectorImages[name],
-                                                          add_to_image=True)
+                if self.apply_sensor_model:
+                    sensor = galsim.SiliconSensor(rng=self._rng,
+                                                  treering_center=detector.tree_rings.center,
+                                                  treering_func=detector.tree_rings.func)
+                    surface_ops = [waves, angles]
+                else:
+                    sensor = None
+                    surface_ops = ()
+
+                # Desired position to draw the object.
```

Comment:
@rmjarvis Here I've updated the code to draw on a postage stamp following your example script.  This speeds up things, of course, but the stamp sizes are sometimes too small for the brighter stars.   Here's an example:
![imsim_draw_on_postage_stamp](https://user-images.githubusercontent.com/1994473/36927361-effde638-1e31-11e8-809a-b86b710001bc.png)
Any suggestions? A different argument to `getGoodImageSize`?